spec/generators/number_generator_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe NumberGenerator do
```

Comment:
hier misschien dan ook direct checken hoe het gaat met de validaties?

Reply:
Dat moeten we dan eigenlijk niet hier checken, maar in integratietests. Hier kun je hooguit checken of iets een min="5" attribuut bevat oid maar dat kan je ook anders schrijven dus lijkt me niet heel zinvol. Ik maak wel wat integratietests.

Reply:
Fixed.